model/prototest/model_test.pb.go

```diff
@@ -3,11 +3,9 @@
 
 package prototest
 
-import (
```

Comment:
this indicates that you're using old plugins. Run `make proto-install` then `make proto`